docs/references/nextjs/trpc.mdx

```diff
@@ -110,11 +123,11 @@ export const exampleRouter = router({
 })
 ```
 
-## Using tRPC Middleware
+## Update your tRPC Middleware
```

Comment:
```suggestion
## tRPC Middleware
```

Reply:
change related to one below

Reply:
Actually, I'd change it to `Create a Protected Procedure`
see my notes one below